pom.xml

```diff
@@ -61,6 +61,11 @@
 	</build>
 	<dependencies>
 		<dependency>
+			<groupId>org.apache.maven</groupId>
```

Comment:
In order to provide the future and previous version of the influxdb (create a database with or without" IF NOT EXISTS") I did a quick version comparison using (org.apache.maven.artifact.versioning.DefaultArtifactVersion).

I could drop this package but version comparison would less elegant...
Or I can just completely drop support for previous version.

What do you prefer ?